src/validators/discussion-validator.js

```diff
@@ -0,0 +1,100 @@
+const  {check, param}  =  require ( 'express-validator' )
+const {StatusCodes} = require ('http-status-codes');
+const {validateResult} = require('../helpers/response-result');
+const {validationId} = require('../validators/account-validator');
+
+
+const validationDiscussionFilters  = [
+    param('filter')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isIn(['news', 'populars', ]).withMessage('El filtro es inválido'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationDiscussionCriterion  = [
+    param('filter')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isIn(['title', 'tracing', ]).withMessage('El filtro es inválido'),
+    param('criterion')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationCriterion = (request, response, next) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const validationCriterion = (request, response, next) => {
```